app/cells/decidim/challenges/challenge_m/show.erb

```diff
@@ -0,0 +1,43 @@
+<div class="card card--challenge <%= resource_state %> muted">
+    <div class="card__content">
+        <div class="card__header">
+            <%= link_to challenge_path, class: "card__link" do %>
+                <div class="card__title">
+                    <%= resource_title %>
+                </div>
+            <% end %>
+        </div>
+        <div class="card__text">
+            <div class="card__text--paragraph">
+                <span class="card__text--paragraph--estat"><%= t(resource_state, scope: 'decidim.challenges.states') %></span><div class="card__text--paragraph--description"><%= description%></div>
+            </div>
+        </div>
+        <div class="card__sdg">
+            <div class="card_text--sdg--image">
+                <div class="ods">
+                    <div class="text-container">
+                        <p><%= t("no_poverty.logo.line1", scope: "decidim.components.sdgs") %></p>
+                        <p><%= t("no_poverty.logo.line2", scope: "decidim.components.sdgs") %></p>
```

Comment:
isn't this hardcoded and should be interpolated by the current SDG?

Reply:
yes, I updated that on the next commit